[15.0][FIX] base_duplicate_security_group: use a proper hook

chienandalu opened the pull request:
cc @Tecnativa TT3142 TT42816

ping @sergio-teruel @CarlosRoca13 @pilarvargas-tecnativa

chienandalu committed:
[FIX] base_duplicate_security_group: use a proper hook

TT3142

CarlosRoca13 commented on base_duplicate_security_group/static/src/js/duplicate.esm.js:
```diff
@@ -11,11 +11,14 @@ import session from "web.session";
+        this._super(...arguments);
+        const base_group = "base_duplicate_security_group.group_duplicate_records";
+        session.user_has_group(base_group).then((result) => {
+            if (!result) {
+                this.controllerParams.activeActions.duplicate = false;
+            }
```
Is `this.controllerParams.activeActions.duplicate` activated without defining it as true?

chienandalu replied:
Yup -> https://github.com/odoo/odoo/blob/1785047d58f28ba11e11e40ba134e89bb235fa8b/addons/web/static/src/legacy/js/views/abstract_view.js#L146

chienandalu commented:
please check @sergio-teruel

pilarvargas-tecnativa committed:
[FIX] base_duplicate_security_group: fix test errors

When the web_responsive module is installed, the tests fail when looking
for "Settings" in the menu, because you have to access the menu first
to get that option. Adding the extra step to display the menu is the
right option so that the tests don't fail whether the web_responsive
module is installed or not.

TT42816

pilarvargas-tecnativa commented:
The tests have been corrected to fix the same problem that occurred with the base_import_security_group module.
PR: https://github.com/OCA/server-ux/pull/665

pedrobaeza reviewed (approved):
/ocabot merge patch
